site/styles/_nav.scss

```diff
@@ -288,12 +288,16 @@
     }
   }
 }
+
 // Header Form Search
 .navbar-nav-search {
   position: absolute;
   left: 160px;
   width: 150px;
   top: $navbar-search-top;
+  @media (max-width: 345px) {
```

Comment:
See my comment elsewhere about arbitrary break points (although this is quite close to the xxs width which is 320px so maybe this could be used here?)